src/looprestoration.rs

```diff
@@ -136,3 +137,160 @@ decl_looprestorationfilter_fns! {
     fn dav1d_wiener_filter7_16bpc_neon,
     fn dav1d_wiener_filter5_16bpc_neon,
 }
+
+// 256 * 1.5 + 3 + 3 = 390
+const REST_UNIT_STRIDE: usize = 390;
+
+// TODO Reuse p when no padding is needed (add and remove lpf pixels in p)
+// TODO Chroma only requires 2 rows of padding.
+// TODO(randomPoison): Temporarily pub until remaining looprestoration fns have
+// been deduplicated.
+#[inline(never)]
+pub(crate) unsafe fn padding<BD: BitDepth>(
+    mut dst: &mut [BD::Pixel; 70 /*(64 + 3 + 3)*/ * REST_UNIT_STRIDE],
```

Comment:
Is it this way in C?  Why not just use `(64 + 3 + 3)` instead `70`?

Reply:
Yeah, this comment is there in a bunch of the C code so I reproduced in Rust.

Reply:
Is it something we want to clean up (when we clean up this function)?  I'm surprised `rustfmt` didn't eat this comment, as it has for other similar ones on array lengths.

Reply:
I was planning to leave the comment in here since it was in the original C. I think we're trying to preserve the C comments as much as possible, right?